src/cmake-tools.ts

```diff
@@ -1164,21 +1164,21 @@ export class CMakeTools implements vscode.Disposable, api.CMakeToolsAPI {
                 const expanded_dest = await expandString(copy_dest, opts);
                 const pardir = path.dirname(expanded_dest);
                 try {
-                    log.debug(localize('copy.compile.commands', 'Copying compile_commands.json from {0} to {1}', compdb_path, expanded_dest));
+                    log.debug(localize('copy.compile.commands', 'Copying {2} from {0} to {1}', compdb_path, expanded_dest, 'compile_commands.json'));
                     await fs.mkdir_p(pardir);
                     try {
                         await fs.copyFile(compdb_path, expanded_dest);
                     } catch (e: any) {
                         // Just display the error. It's the best we can do.
-                        void vscode.window.showErrorMessage(localize('failed.to.copy', 'Failed to copy "{0}" to "{1}": {2}', compdb_path, expanded_dest, e.toString()));
+                        void vscode.window.showErrorMessage(localize('failed.to.copy', 'Failed to copy {0} to {1}: {2}', `"${compdb_path}"`, `"${expanded_dest}"`, e.toString()));
                     }
                 } catch (e: any) {
                     void vscode.window.showErrorMessage(localize('failed.to.create.parent.directory',
-                        'Tried to copy "{0}" to "{1}", but failed to create the parent directory "{2}": {3}',
-                        compdb_path, expanded_dest, pardir, e.toString()));
+                        'Tried to copy {0} to {1}, but failed to create the parent directory {2}: {3}',
+                        `"${compdb_path}"`, `"${expanded_dest}"`, `"${pardir}"`, e.toString()));
                 }
             } else {
-                log.debug(localize('cannot.copy.compile.commands', 'Cannot copy compile_commands.json because it does not exist at {0}', compdb_path));
+                log.debug(localize('cannot.copy.compile.commands', 'Cannot copy {1} because it does not exist at {0}', compdb_path, 'compile_commands.json'));
```

Comment:
compdb_path does this need to be changed too?

Reply:
`comp_db` is already separated out and substituted as part of the `localize` call.  Or am I missing something?  I didn't add quotes here because it didn't already have them.